djangocms_versioning/static/djangocms_versioning/js/create_url.js

```diff
@@ -0,0 +1,71 @@
+(function($) {
```

Comment:
We don't have the BE content type widget, I guess it would be nice to supply the widget but I was hoping to change this implementation in the future where a package can supply its widget to a content type search field.